src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1042,7 +1042,7 @@
     <value>Run at startup</value>
   </data>
   <data name="GeneralPage_RunAtStartUp.Description" xml:space="preserve">
-    <value>PowerToys will launch automatically</value>
+    <value>PowerToys will open automatically</value>
```

Comment:
Shouldn't run here be better? Thinking we're not actually opening any window or anything like that.
```suggestion
    <value>PowerToys will run automatically</value>
```

Reply:
"Technically" opening an app is independent from to opening a window. But I see that, in practice, one might _expect_ a window... By that, both are possible, but I can use "run" here...